beacon-chain/rpc/beacon/attestations.go

```diff
@@ -274,39 +274,29 @@ func (bs *Server) StreamIndexedAttestations(
 				}
 				bs.ReceivedAttestationsBuffer <- data.Attestation.Aggregate
 			}
-		case atts, ok := <-bs.CollectedAttestationsBuffer:
+		case aggAtts, ok := <-bs.CollectedAttestationsBuffer:
 			if !ok {
 				log.Error("Indexed attestations stream collected attestations channel closed")
 			}
-			// We aggregate the received attestations.
-			aggAtts, err := helpers.AggregateAttestations(atts)
-			if err != nil {
-				return status.Errorf(
-					codes.Internal,
-					"Could not aggregate attestations: %v",
-					err,
-				)
-			}
-			if len(aggAtts) == 0 {
-				continue
-			}
+
 			// All attestations we receive have the same target epoch given they
 			// have the same data root, so we just use the target epoch from
 			// the first one to determine committees for converting into indexed
 			// form.
-			epoch := aggAtts[0].Data.Target.Epoch
-			committeesBySlot, _, err := bs.retrieveCommitteesForEpoch(stream.Context(), epoch)
+			targetRoot := aggAtts[0].Data.Target.Root
+			targetEpoch := aggAtts[0].Data.Target.Epoch
+			committeesBySlot, _, err := bs.retrieveCommitteesForRoot(stream.Context(), targetRoot)
```

Comment:
s/committeesBySlot/committeesByRoot

Reply:
It's still committeesBySlot since its a map of slot -> committee